homeassistant/components/mercedesme.py

```diff
@@ -136,6 +144,8 @@ def async_added_to_hass(self):
 
     def _update_callback(self):
         """Callback update method."""
+        # If the method is made a callback this should be changed

```

Comment:
I personally don't like todo items in the source but I don't know if there is a rule against it.